dev/modules/base-model.js

```diff
@@ -148,8 +148,8 @@ class RestBaseModel {
           );
           addCustomHeaders(request);
           request.exec()
-            .then((response) => { resolve({ response, request: request.xhr }); })
-            .catch((response) => { reject({ response, request: request.xhr }); });
+            .then((response) => { resolve({ response, request: request.xhr })})
```

Comment:
Please rebase before starting development and double check the code you commit before opening a pull request...

lines: [151, 152]